include/irr/asset/IDescriptor.h

```diff
@@ -1,11 +1,13 @@
 #ifndef __IRR_I_DESCRIPTOR_H_INCLUDED__
 #define __IRR_I_DESCRIPTOR_H_INCLUDED__
 
+#include "irr/core/IReferenceCounted.h"
+
 namespace irr {
 namespace asset
 {
 
-class IDescriptor
+class IDescriptor : public virtual core::IReferenceCounted
```

Comment:
not sure there's a need for this.

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/387#issuecomment-546553793